### DISCUSSION TITLE
发现一个比较好的Mac 终端 —— warp

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#124] [Show and tell]
直接说优点
1. rust 写的，快，且安全
2. 每次交互都会生成一个block，可以对这个block进行标记等操作。输出格式友好，还会自动输出每个命令的执行时间
![image](https://user-images.githubusercontent.com/77786952/194749094-14d7915a-6b48-48bb-84b5-6ec729c3044a.png)
3. 相比于 nvim 界面，感觉UI更加“锐利”一些
4. 使用 nvim 的时候，相比于iterm2，首先，它不会漏出后面的终端背景。
![image](https://user-images.githubusercontent.com/77786952/194749182-26dc3b83-3eee-4f66-8b04-f6728b0c16e5.png)
其次，warp中可以使用鼠标进行一些辅助操作。比如鼠标滚轮滚动，只会在鼠标所在窗口滚动，不会导致整个nvim界面以及其他窗口滚动；鼠标拖动选中的时候不会选中到其他窗口的同行。

iterm 
![image](https://user-images.githubusercontent.com/77786952/194749325-2a22b48c-4b98-4eae-8625-c7f6e6c8d54a.png)

warp
![image](https://user-images.githubusercontent.com/77786952/194749340-1a1d1158-df1f-41ce-a423-18fef59df70c.png)

今天是第一天使用，感觉非常优秀

**Comment by Linya-IronMan** [comment]
它号称是最像 code editor 的 terminal

**Comment by nshen** [comment]
wrap是不是需要登录才能用的那个啊

**Reply by Linya-IronMan** [reply]
是的

**Comment by Linya-IronMan** [comment]
缺点可能就是要登录，感觉之后可能要收费。但是现在用一用没啥问题

**Reply by nshen** [reply]
我觉得一个终端也要登录太扯淡了。功能再好也先不用，haha
